### ISSUE TITLE
Contracts on 0 after update to 2.1.2

### ISSUE THREAD

**Issue by rooie1971** [bittorrent/go-btfs#153]
Hi,

I've updated to 2.1.2 on linux, and in the dashboard I see that there are 0 contracts, before updating there were a lot  more contracts is this a dashboard glitch/bug?

Regards,
Richard

**Comment by Sophia-yu-kk** [comment]
From which version did you upgrade to V2.1.2?
Please provide us with your node id.thanks

**Comment by rooie1971** [comment]
upgraded from 2.1.1
Peer identity: 16Uiu2HAmPZaoLN3E9ohD4wzABDTx8owBTpFgZAUc8GpBpK3iTuhM
Peer identity: 16Uiu2HAmGn28DbjjX323fgBV74iUddSsaESWazYfN8sBcYFyupWQ

And also it seems the memory leak is still present, using 64bit Linux Ubuntu as a host.

**Comment by Sophia-yu-kk** [comment]
btfs storage contracts sync host
btfs storage contracts list host
Please execute these two commands to see the results returned.

**Comment by rooie1971** [comment]
The contracts are visible now, without me changing anything, weird. Only old earnings are not there, uncashed amount had btt in it, and shows as 0 now.

**Comment by rooie1971** [comment]
Let me rephrase that, when looking at the dashboard on chrome it displays 0 contracts, when looking at the same dashboard page on firefox it displays correct. But on both browsers the uncashed amount is 0.
The list contracts command show the contracts by the way.

**Comment by pkyriacou** [comment]
> The contracts are visible now, without me changing anything, weird. Only old earnings are not there, uncashed amount had btt in it, and shows as 0 now.

Please note that the 2.1.2 upgrade instructions directed us to cash any uncashed cheques, as these would remain on the old vault address.

**Comment by Sophia-yu-kk** [comment]
To cash a check before upgrading to V2.1.2, please refer to the tutorial at https://docs.btfs.io/docs/tutorials-for-upgrading-to-v212
Also, please clear your browser cache to see the next.
